doc/iterations/2023.6/shapes/shape Migration flow.adoc

```diff
@@ -0,0 +1,29 @@
+= (appetite) Shape
+
+== Problem
+
+In order to include a test which covers the all odesign to view migration path, we will migrate the existing flow model.
+
+This view model should be generated using the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
```

Comment:
You need to specify clearly the order of the various tasks that you will perform during this iteration. It's not clear for me yet and there are multiple possible solutions.